README.adoc

```diff
@@ -14,8 +14,11 @@ This repository contains:
 * Local Development environment
 
 == Getting Started
+=== Installation
 
-link:https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/local-setup.adoc[Mobile installation guide]
+link:https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/local-setup.adoc[Ansible based installer] Sets up mobile on your host machine.
```

Comment:
It could be interpreted as 2 steps. If you do not mind to add something like this:
`Mobile on OpenShift can be installed using Ansible based installer and Minishift vm.`